editor.planx.uk/src/pages/FlowEditor/components/Sidebar/Publish/PublishFlowButton.tsx

```diff
@@ -52,6 +52,8 @@ export const PublishFlowButton: React.FC<{ previewURL: string }> = ({
     try {
       setLastPublishedTitle("Checking for changes...");
       const alteredFlow = await validateAndDiffFlow(flowId);
+      console.log(alteredFlow);
```

Comment:
Another `console.log` here - again, please take a quick scan through your own code before opening for review - I want to focus on the juicy changes only :slightly_smiling_face: